docs/tutorials/external_mqtt_clients.md

```diff
@@ -0,0 +1,74 @@
+
+# Prerequisite
+Before you start you need have access to a running Kubernetes cluster environment. A [Minikube](https://minikube.sigs.k8s.io/docs/start/) with [Ingress](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/) running on your laptop will just do fine. The ArtemisCloud operator also runs in a Openshift cluster environment like [CodeReady Container](https://developers.redhat.com/products/codeready-containers/overview). In this blog we assume you have Kubernetes cluster environment. Execute the following command to enable Ingress in minikube:
+
+```shell script
+$ minikube addons enable ingress
+```
+
+# Enable SSL Passthrough
+[SSL Passthrough](https://kubernetes.github.io/ingress-nginx/user-guide/tls/) leverages [SNI](https://en.wikipedia.org/wiki/Server_Name_Indication) and reads the virtual domain from the TLS negotiation, which requires compatible clients. After a connection has been accepted by the TLS listener, it is handled by the controller itself and piped back and forth between the backend and the client. Execute the following command to enable SSL Passthrough in minikube:
+
+```shell script
+$ minikube kubectl -- patch deployment -n ingress-nginx ingress-nginx-controller --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value":"--enable-ssl-passthrough"}]'
+```
+
+# Deploy ArtemisCloud operator
+First you need to deploy the ArtemisCloud operator.
+If you are not sure how to deploy the operator take a look at [this blog]({{< relref "using_operator.md" >}}).
+
+# Download the test certficates from Apache ActiveMQ Artemis
+```shell script
+$ wget -O server-keystore.jks https://github.com/apache/activemq-artemis/raw/main/tests/security-resources/server-keystore.jks
+$ wget -O client-ca-truststore.jks https://github.com/apache/activemq-artemis/raw/main/tests/security-resources/client-ca-truststore.jks
+$ wget -O server-ca-keystore.p12 https://github.com/apache/activemq-artemis/raw/main/tests/security-resources/server-ca-keystore.p12
+$ keytool -storetype pkcs12 -keystore server-ca-keystore.p12 -storepass securepass -alias server-ca -exportcert -rfc > server-ca.crt
+```
+
+# Create a secret with the test certificates
+```shell script
+$ kubectl create secret generic my-tls-secret \
+--from-file=broker.ks=server-keystore.jks \
+--from-file=client.ts=client-ca-truststore.jks \
+--from-literal=keyStorePassword=securepass \
+--from-literal=trustStorePassword=securepass
+```
+
+### Deploy ActiveMQArtemis with an mqtt acceptor
+```shell script
+$ kubectl apply -f - <<EOF
+apiVersion: broker.amq.io/v1beta1
+kind: ActiveMQArtemis
+metadata:
+  name: artemis-mqtt-ssl
+spec:
+  acceptors:
+    - name: my-acceptor
+      expose: true
+      port: 5672
+      protocols: mqtt
+      sslEnabled: true
+      sslSecret: my-tls-secret
+  env:
+    - name: JAVA_ARGS_APPEND
+      value: -Djavax.net.debug=all
+EOF
+```
+
+### Publish a message with mosquitto_pub
```

Comment:
Wouldn't be helpful to provide some get/download link to image/mosquito_pub app? As it's tutorial, I am stupid and can't use google properly. Would prefer to just click on some useful link. :-)

https://mosquitto.org/download/ ?

Reply:
Done, I also added the command to publish a message with mosquitto_pub from the eclipse-mosquitto container.